### DISCUSSION TITLE
2024 4Q <성장 수집가> 공준호의 OKR

### DISCUSSION THREAD

**Discussion by Generoustandard** [Pseudo-Lab/growth-collector#20] [우리들의 성장 목표!]
O: 직무에 맞는 부서 이동 및 지속 성장
KR1: SQL 등 데이터 베이스 관련 공부
KR2: 데이터분석, ML/DL 관련 공부
KR3: 언어 공부 및 지속적 교류

**Comment by Generoustandard** [comment]
Action Item 2024/11/29 ~ 2024/12/05

1. Elice SQL 기초 수강
2. Udemy 인공지능 마스터 클래스 수강
3. 일본어 공부 및 일본에서 일본어 많이 사용하기

**Comment by Generoustandard** [comment]
Action Item 2024/12/06 ~ 2024/12/12

1. Elice SQL 복습 및 프로그래머스 코테 연습
2. Udemy 인공지능 마스터 클래스 & mySUNI 딥러닝 수학 강의 공부
3. 영어 복습 및 일본어 공부

**Comment by Generoustandard** [comment]
Action Item 2024/12/13 ~ 2024/12/19

1. Elice SQL 복습 및 프로그래머스 코테 연습
2. Udemy 인공지능 마스터 클래스 & mySUNI 딥러닝 수학 강의 공부
3. 영어 복습 및 일본어 공부
4. 12월 연말 유럽 여행 계획 수립

**Reply by Generoustandard** [reply]
지난 한 주는 비슷하게 흘러갔던 것 같습니다~
14박 17일정도로 유럽 여행을 갈 계획이라 12월달 마법학교 참석은 힘들겠네요 😢

**Reply by rlaalsrl0922** [reply]
헉 유럽 어디가시나요!!

**Reply by soncy0311** [reply]
여행 무탈히 즐겁게 다녀오세요 ㅎㅎ

**Reply by Generoustandard** [reply]
> 헉 유럽 어디가시나요!!

답장하신거 지금 봤네요! 이탈리아, 스위스, 프랑스, 영국이요~

**Reply by Generoustandard** [reply]
> 여행 무탈히 즐겁게 다녀오세요 ㅎㅎ

감사합니다 ㅎㅎ
